### PR TITLE
Switch from "require" syntax to "import" where possible

### PR DESCRIPTION
**Pull request by ncksllvn**
This PR switches from `require()` to `import` where possible. Exceptions include conditional-requires, because I don't believe that's possible yet with `import`. For consistency I also moved imports that do not contain an export to the top of each file.

Resolves department-of-veterans-affairs/vets.gov-team/issues/2944

### CODE REVIEW

**Comment by ncksllvn** [comment]
Ugh, Code Climate is giving an error about repetitive code. Weren't we talking about removing that?

**Comment by jbalboni** [comment]
It shouldn't block merging, once someone approves this PR

**Review by cvalarida — APPROVED** [review]
I'll admit my ignorance of how `import`ing SCSS works, but assuming you tested that manually (or using visual regression testing!) and did a search for `require(` in all the JS, I'd call this good!

**Review by jbalboni — APPROVED** [review]

File: src/js/health-beta/health-beta-entry.jsx
Comment (U-DON) [inline src/js/health-beta/health-beta-entry.jsx]: I don't think styles need to be imported for this app.

**Review by U-DON — APPROVED** [review]
LGTM.

File: src/js/health-beta/health-beta-entry.jsx
Comment (ncksllvn) [reply src/js/health-beta/health-beta-entry.jsx]: So I can safely remove those 3 lines?

**Comment by ncksllvn** [comment]
Thanks @cvalarida, @jbalboni, and @U-DON!

File: src/js/health-beta/health-beta-entry.jsx
Comment (U-DON) [reply src/js/health-beta/health-beta-entry.jsx]: Yeah, I don't think those two lines are used.

Comment (ncksllvn) [reply src/js/health-beta/health-beta-entry.jsx]: Ah I would have removed it but I'm gonna leave it to whoever maintains that just in case there are some styles being used out of there for a modal or something. I'm not familiar with that app at all.

**Pull request merged**
